x-pack/test/api_integration/apis/ml/utils.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export function compareByFieldName(a: { fieldName: string }, b: { fieldName: string }) {
```

Comment:
Could you reduce some of the code duplication in here by adding a generic `sort` function which calls something like `stringA.localeCompare(stringB)`? Otherwise potentially we'll have to keep dupicating the logic if we want to sort objects by another property like `name`.

Reply:
agree, this function can receive 3 arguments: `objA`, `objB`, and `fieldName` - the name of the field we want to use for comparison, e.g. `name`, `fieldName`

Reply:
and you can always use [sortBy from lodash](https://lodash.com/docs/4.17.15#sortBy)

Reply:
Changed to lodash sortBy in 86a695d